packages/site/playwright.config.ts

```diff
@@ -0,0 +1,47 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test';
+import path from 'path';
+
+// Use process.env.PORT by default and fallback to port 3000
+const PORT = process.env.PORT || 3000;
+
+// Set webServer.url and use.baseURL with the location of the WebServer respecting the correct set port
+const baseURL = process.env.PLAYWRIGHT_TEST_BASE_URL || `http://localhost:${PORT}`;
+
+// Reference: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  // Timeout per test
+  timeout: 10 * 1000,
+  // Test directory
+  testDir: path.join(__dirname, 'e2e'),
+  // If a test fails, retry it additional 2 times
+  retries: 2,
+  // Artifacts folder where screenshots, videos, and traces are stored.
+  outputDir: 'test-results/',
+
```

Comment:
what’s our thoughts on storage 

maybe for now we can just use without screenshots

Reply:
could also use chromatic

Reply:
This looks to be the most feasible option for image storage https://git-lfs.com/